pkg/utils/utils_test.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import (
+	"testing"
+)
+
+func TestGetClusterNameFromKubectlCmd(t *testing.T) {
+
+	type test struct {
+		input    string
+		expected string
+	}
+
+	tests := []test{
+		{input: "get pods --cluster-name=minikube", expected: "minikube"},
+		{input: "--cluster-name minikube1", expected: "minikube1"},
```

Comment:
Can we try `--cluster-nameminikube1` ?

Reply:
yes added and passed 
{input: "--cluster-nameminikube1", expected: ""},